cargo-apk/README.md

```diff
@@ -151,6 +151,12 @@ icon = "@mipmap/ic_launcher"
 # Defaults to the compiled artifact's name.
 label = "Application Name"
 
+# See https://developer.android.com/guide/topics/manifest/application-element#extractNativeLibs
+extract_native_libs = true
```

Comment:
In hindsight we should have also used this property to steer whether libs are compiled in the APK

https://github.com/rust-mobile/cargo-apk/blob/8ce097b2b40e2ba0f699085a1c0f3b5c950254e5/ndk-build/src/apk.rs#L203-L205

Per the documentation linked for this property, libraries are stored uncompressed when they are _not_ extracted by the installer. Right now I think one can create an invalid APK when setting `extract_native_libs = false` and building a non-debug (release) profile, which sets `disable_aapt_compression = false`:

https://github.com/rust-mobile/cargo-apk/blob/8ce097b2b40e2ba0f699085a1c0f3b5c950254e5/cargo-apk/src/apk.rs#L214